docs/docs/naming.md

```diff
@@ -44,9 +74,9 @@ $ dig _search._tcp.marathon.mesos SRV
 
 ;; ANSWER SECTION:
 _search._tcp.marathon.mesos.	60 IN SRV 0 0 31302 10.254.132.41.
-``` 
+```
 
-SRV records are generated only for tasks that have been allocated a specific port through Mesos. 
+*Note*: SRV records are generated only for tasks that have been allocated a specific port through Mesos and that have no container IP assigned.
```

Comment:
SRV records are generated for all tasks that have allocated host port resources through Mesos.

In the future, additional SRV records will be generated for tasks that provide DiscoveryInfo.